src/Kiota.Builder/CodeDOM/CodeNamespace.cs

```diff
@@ -29,6 +30,18 @@ public override string Name
             StartBlock.Name = name;
         }
     }
+
+    public CodeFile FindFileOrInitializeWith(string fileName, params CodeElement[] children)
+    {
+        if (FindChildByName<CodeFile>(fileName, false) is CodeFile existingFile)
+            return existingFile;
```

Comment:
I'm guessing the intent would be to add the children to the existing code file here before we return it?